packages/core/lib/match.js

```diff
@@ -23,20 +21,13 @@ function match (test, options, configSetters) {
 
   assertConfigSetters(configSetters)
 
-  const match = { test: createFileTypeMatcher(test) }
+  options.test = createFileTypeMatcher(test)
```

Comment:
We should print a warning if `test` was set in options.

Reply:
Yeah, if `options.test` was set before this line 👍